aict_tools/scripts/train_dxdy_regressor.py

```diff
@@ -82,8 +82,7 @@ def main(configuration_path, signal_path, predictions_path, dxdy_model_path, key
 
     log.info('Events after nan-dropping: {} '.format(len(df_train)))
 
-    target_dx = df['true_dx'].loc[df_train.index]
-    target_dy = df['true_dy'].loc[df_train.index]
+    target_dxdy = df.loc[df_train.index, ['true_dx', 'true_dy']]
```

Comment:
add `.to_numpy()` here to convert to numpy array directly

Reply:
first got the error that the numpy array doesn't have the attribute 'values'. I changed it the way you mentioned it and it's working fine.